CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java

```diff
@@ -151,6 +173,16 @@ protected boolean providerDetection() {
 				return options.hasOption("providerDetection");
 			}
 			
+			@Override
+			protected List<String> getIgnorePackageList(){
+				return ignorePackageList;
+			}
+			
+			@Override
+			protected boolean ignorePackagesflag() {
```

Comment:
also i think this whole property should be a computed porperty where it is only true when `getIgnoredPackages` has any elements

Reply:
ignorePackagesflag() has been deleted. instead of using a flag, updated the code with checking if getIgnoredPackages has any elements